lte/gateway/c/oai/tasks/nas/emm/sap/emm_cn.c

```diff
@@ -637,6 +636,9 @@ static int _emm_cn_pdn_connectivity_res(emm_cn_pdn_res_t *msg_pP)
 
     OAILOG_DEBUG(LOG_NAS_EMM, "ESM encoded MSG size %d\n", size);
 
+    if (msg_pP->pdn_addr) {
+      bdestroy_wrapper(&msg_pP->pdn_addr);
```

Comment:
bdestroy_wrapper check whether msg_pP->pdn_addr is Null or not, so no need for the check in line 639.